FreeRTOS-Plus/Demo/AWS/Device_Defender_Windows_Simulator/Device_Defender_Demo/DemoTasks/DefenderDemoExample.c

```diff
@@ -291,20 +291,6 @@ static bool prvPublishDeviceMetricsReport( size_t xReportLength );
 static bool prvValidateDefenderResponse( const char * pcDefenderResponse,
                                          size_t xDefenderResponseLength );
 
-/**
- * @brief The task used to demonstrate the Defender API.
- *
- * This task collects metrics from the device using the functions in
- * metrics_collector.h and uses them to build a defender report using functions
- * in report_builder.h. Metrics include the number for bytes written and read
- * over the network, open TCP and UDP ports, and open TCP sockets. The
- * generated report is then published to the AWS IoT Device Defender service.
- *
- * @param[in] pvParameters Parameters as passed at the time of task creation.
- * Not used in this example.
- */
-static void prvDefenderDemoTask( void * pvParameters );
```

Comment:
Why was this declaration removed? The function is still present

Reply:
It's no longer a static function.